donate/index.html

```diff
@@ -42,38 +42,25 @@ <h3 class="text-center">Want to keep NewPipe running?</h3>
                     <div class="col-sm-offset-4">
                         <div class="row" id="sorted-row">
                             <div class="col-sm-12" id="donate-title">
-                                <h3 class="with-subtitle">What are you waiting for?</h3>
+                                <h3 class="with-subtitle">Do you like what we did so far?</h3>
                                 <p class="subtitle">Help us to bring NewPipe forward!</p>
                             </div>
                             <div class="col-xs-12 col-lg-10" id="button-container">
-                                <div class="button-wrapper" id="bitcoin-donation-button">
-                                    <button class="button button-large action donate">Bitcoin</button>
-                                    <div class="clearfix"></div>
-                                    <span>one-time donation</span>
-                                </div>
-                                <div class="button-wrapper" id="bountysource-donation">
-                                    <a href="https://www.bountysource.com/teams/newpipe/" target="_blank">
-                                        <button class="button button-large action donate">Bountysource</button>
-                                    </a>
-                                    <div class="clearfix"></div>
-                                    <span>boost an upcoming feature</span>
-                                </div>
                                 <div class="button-wrapper" id="liberapay-donation">
                                     <a href="https://liberapay.com/TeamNewPipe/" target="_blank">
                                         <button class="button button-large action donate">Liberapay</button>
                                     </a>
                                     <div class="clearfix"></div>
                                     <span>recurrent donations&nbsp;&nbsp;</span>
                                 </div>
-                            </div>
-                            <div class="col-sm-12 col-xs-12 col-lg-10" id="bitcoin-donation">
-                                <p>
-                                    <a href="bitcoin:16A9J59ahMRqkLSZjhYj33n9j3fMztFxnh">
-                                        <img src="{{ site.baseurl }}/img/bitcoin_address.png" alt="bitcoin qr code" />
-                                    </a>
-                                </p>
-                                <p>Please Donate To Bitcoin Address: </p>
-                                <samp>16A9J59ahMRqkLSZjhYj33n9j3fMztFxnh</samp>
+                                <div class="button-wrapper" id="bank-transfer-donation">
+<pre>
+NewPipe e.V.
+Bank name
```

Comment:
This clearly isn't ready yet.

Reply:
That part is hidden for now, but I removed it anyway.